source/app/controllers/index.rb

```diff
@@ -1,3 +1,31 @@
 get '/' do
   erb :index
 end
+
+get '/game' do
+  @user1 = User.first
+  @user2 = User.last
+  erb :gameboard
+end
+
+post '/game' do
+  @current_game = RockPaperScissors.new(params)
+  @winner,@loser,@winning_token,@losing_token = @current_game.determine_winner
```

Comment:
This is a little confusing to me. Why are you assigning all the variables at once.